unit-tests/dds/test-options.py

```diff
@@ -0,0 +1,103 @@
+# License: Apache 2.0. See LICENSE file in root directory.
+# Copyright(c) 2022 Intel Corporation. All Rights Reserved.
+
+#test:donotrun:!dds
+
+import pyrealdds as dds
+from rspy import log, test
+
+
+dds.debug( True, 'C  ' )
+log.nested = 'C  '
+
+
+participant = dds.participant()
+participant.init( 123, "test-options" )
+
+info = dds.device_info()
+info.name = "Test Device"
+info.topic_root = "realdds/device/topic-root"
+
+
+import os.path
+cwd = os.path.dirname(os.path.realpath(__file__))
+remote_script = os.path.join( cwd, 'options-server.py' )
+with test.remote( remote_script, nested_indent="  S" ) as remote:
+    remote.wait_until_ready()
+    #
+    #############################################################################################
+    #
+    test.start( "Test no options..." )
```

Comment:
No need to add ... on every test postfix - IMO

Reply:
Done